app/src/main/java/fr/free/nrw/commons/nearby/NearbyController.java

```diff
@@ -98,21 +100,24 @@ public int compare(Place lhs, Place rhs) {
 
         placeList = placeList.subList(0, Math.min(placeList.size(), MAX_RESULTS));
 
+        Bitmap icon = UiUtils.getBitmap(
+                VectorDrawableCompat.create(
```

Comment:
@tobias47n9e I believe you wrote NearbyControllerTest. Perhaps could you figure out what is happening here?